src/fmu/sumo/explorer/objects/case.py

```diff
@@ -16,8 +16,10 @@
 class Case(Document):
     """Class for representing a case in Sumo"""
 
-    def __init__(self, sumo: SumoClient, metadata: Dict, pit: Pit = None):
+    def __init__(self, sumo: SumoClient, metadata: Dict, summary: Dict,
```

Comment:
The name `summary` may be confusing, as the term is quite established for some specific data produced by simulators. Other synonyms that can be considered?